blockchain/for-companies/how-to-run-liberland-company.md

```diff
@@ -0,0 +1,132 @@
+## Running Liberland registered companies
+So you have decided to pioneer liberty and decentralization by opening a company and doing business in Liberland.
+
+This article will explain how to create a company or open a subsidiary of another company, 
```

Comment:
(create a company..)... to operate in Liberland's blockchain system and economy...